tests/foreman/cli/test_environment.py

```diff
@@ -242,18 +241,13 @@ def test_positive_sc_params_by_id(self):
 
         @Assert: The command runs without raising an error
 
-        @BZ: 1219934
-
         """
         environment = make_environment()
         Environment.sc_params({
             'environment-id': environment['id'],
         })
-        if not bz_bug_is_open(1219934):
-            self.fail('BZ #1219934 is closed, should assert the content')
```

Comment:
I've tried to implement this assert but I was not able to get the content in a proper format because `Environment.sc_param` is a subcommand and its output is unstructured plain text from command stdout.

```python
ipdb> environment
{u'updated-at': u'2016/04/19 22:22:24', u'puppetclasses': {}, u'created-at': u'2016/04/19 22:22:24', u'id': u'7', u'name': u'Rgjn9s'}

ipdb> output = Environment.sc_params({'environment-id': environment['id']})
pdb> output
[u'---|-----------|---------------|----------|-------------', u'ID | PARAMETER | DEFAULT VALUE | OVERRIDE | PUPPET CLASS', u'---|-----------|---------------|----------|-------------', u'']
```

I would implement this as 

```python
for fieldname in (u"ID", u"PARAMETER", ....):
    self.assertIn(fieldname, output[1].split(" | "))
```

But I do not like that solution, maybe there is built-in better way to get that data or develop a better error-prone-less parser or even questioning if this is really necessary.

IMO: Not necessary to check the content as the test is expecting a raising exception.